### ISSUE TITLE
Incorporating slope orientation (aspect) as an HRU variable 

### ISSUE THREAD

**Issue by Musselman** [NCAR/summa#146]
Accurate slope-scale simulations in complex terrain require terrain orientation to be explicitly considered in the calculation of solar insolation. I've spoken to Marytn about this issue. It seems that aspect is currently a hardwired variable in engine/sunGeomtry.f90 and engine/derivforce.f90 (specified as 0) and should be brought to the front end and incorporated as an HRU-specific variable similar to slope or LAI.

**Comment by ashleymedin** [comment]
I think we have this now @wknoben

**Comment by andywood** [comment]
I recall writing some code in SUMMA around this years ago when doing our
watershed discretization work.  Backward compatible so that if you left it
out of the attributes, you still could run a flat earth model.

**Comment by ashleymedin** [comment]
Yes, it's here @andywood, @wknoben https://github.com/CH-Earth/summa/commit/657151d828cbddfe0dbf53f3b38e8b4bb4989d62#diff-a3b223dbb9cf9fc96d06048f80696f238eedd6c5a4b3f631b788e139381aa517

**Comment by wknoben** [comment]
OK, so to summarize, if `aspect` not provided in the attributes it runs flat earth, otherwise it uses aspect for the `azimuth` variable and derives `slope` from `tan_slope`.  See related new docs issue: https://github.com/CH-Earth/summa/issues/625

**Comment by ashleymedin** [comment]
Yes, agreed.
